conans/test/unittests/client/build/cmake_test.py

```diff
@@ -728,7 +728,7 @@ def check(text, build_config, generator=None, set_cmake_flags=False):
 
         settings.compiler.version = "15"
         settings.arch = "armv8"
-        check('-G "Visual Studio 15 2017 ARM" -DCONAN_IN_LOCAL_CACHE="OFF" '
+        check('-G "Visual Studio 15 2017" -A "ARM64" -DCONAN_IN_LOCAL_CACHE="OFF" '
```

Comment:
👀

Reply:
According to our existing implementation, `armv8` should be translated into `ARM64`, and according to CMake this platform is not supported using `-G` argument (https://cmake.org/cmake/help/v3.16/generator/Visual%20Studio%2015%202017.html)